app/src/main/java/org/fossasia/pslab/SearchModel.java

```diff
@@ -0,0 +1,27 @@
+package org.fossasia.pslab;
+
+import ir.mirrajabi.searchdialog.core.Searchable;
+
+/**
+ * Created by Harsh on 07-05-2018.
+ */
+
+public class SearchModel implements Searchable {
+
+    private String mTitle;
+
+    public SearchModel(String mTitle)
+    {
```

Comment:
Use these brackets inline with the method title. Similar to `getTitle()`. Try to run a auto code format in Android studio and it will fix all these issues.